apisix/upstream.lua

```diff
@@ -20,11 +20,13 @@ local discovery = require("apisix.discovery.init").discovery
 local upstream_util = require("apisix.utils.upstream")
 local apisix_ssl = require("apisix.ssl")
 local events = require("apisix.events")
+
 local error = error
 local tostring = tostring
 local ipairs = ipairs
 local pairs = pairs
 local pcall = pcall
+local table_insert = table.insert
```

Comment:
no changes required

Reply:
`upstream.lua` also contains invalid changes, if we remove it we will simplify the PR.

https://github.com/apache/apisix/pull/11499/files#diff-9af33577e041ece8fef32d8f8193f46eac83dab9e947a2cc940f36b83d5150e8R23

Reply:
whoops, fixed it.